pdk/1.0/modules/ROOT/pages/policies-pdk-compile-policies.adoc

```diff
@@ -0,0 +1,41 @@
+= Compiling Custom Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+You can compile the custom policy project after you complete all the steps in xref:policies-pdk-create-project.adoc[].
+
+You must compile the custom policy before you can deploy it to a Flex Gateway.
+
+After you compile your project, test your policy with the following tutorials:
+
+* xref:policies-pdk-debug-local.adoc[]
+* xref:policies-pdk-apply-policies.adoc[]
+
+== Before You Begin
+
+Ensure you have completed all the steps in xref:policies-pdk-create-project.adoc[].
+
+== Build Command
+
+To compile the policy, run the `build` command from the policy's root folder:
```

Comment:
```suggestion
To compile the policy, run the `make build` command from the policy's root folder:
```
Maybe include "make" so that it's crystal-clear? At first read, as a newbie, I thought that maybe the build command was something different from the build command in make. It's made apparent soon enough below, but I was still a little confused.